nep-11.mediawiki

```diff
@@ -0,0 +1,176 @@
+<pre>
+  NEP: 11
+  Title: Non-Fungible Token Standard
+  Author: Joe Stewart <hal0x2328@splyse.tech>, Shane Mann <fetter@splyse.tech>
+  Type: Standard
+  Status: Accepted
+  Created: 2018-04-18
+</pre>
+
+==Abstract==
+
+This NEP defines a standard non-fungible token system for the NEO Smart Economy.
+
+==Motivation==
+
+NFTs are required to track, exchange and enforce ownership of digital assets. A non-fungible token (NFT) can be thought of like a property deed - each one is unique and carries some non-mutable information (e.g. the physical address of the property). Other information, such as the owner of the property, can be changed. Also, we provide a built-in optional divisibility within each non-fungible asset. This allows for high value objects to be tokenized more effectively.
+
+==Specification==
+
+===Methods===
+
+====name====
+
+<pre>
+public static string name()
+</pre>
+
+Returns the name of the token. e.g. <code>"MyNFT"</code>. 
+
+This method MUST always return the same value every time it is invoked. 
+
+====symbol====
+
+<pre>
+public static string symbol()
+</pre>
+
+Returns a short string symbol of the token managed in this contract. e.g. <code>"MNFT"</code>. This symbol SHOULD be short (3-8 characters is recommended), with no whitespace characters or new-lines and SHOULD be limited to the uppercase latin alphabet (i.e. the 26 letters used in English). 
+
+This method MUST always return the same value every time it is invoked. 
+
+====totalSupply====
+
+<pre>
+public static BigInteger totalSupply()
```

Comment:
For example, if we are talking about all cars of a state in Brazil, then, `totalSupply` would return that amount that can increase or decrease based on an entity that is registering that assets, right?